README.md

```diff
@@ -83,6 +83,8 @@ The ps library is compatible with all AIX format descriptors of the ps command-l
   - The corresponding host PID of a container process.
 - **huser**
   - The corresponding effective user of a container process on the host.
+- **huid**
```

Comment:
Also need one for `uid` don't we?

Reply:
@vrothberg in `README.md` we can find:
```
The ps library is compatible with all AIX format descriptors of the ps command-line utility (see man 1 ps for details) but it also supports some additional descriptors that can be useful when seeking specific process-related information.
```
so I put only description for `huid` as `uid` is not additional and we can see it in standard `ps` command. WDYT?

Reply:
OK, sounds good, thanks!